src/org/opendatakit/briefcase/ui/reused/SourceDisplayPanelForm.java

```diff
@@ -0,0 +1,94 @@
+package org.opendatakit.briefcase.ui.reused;
+
+import java.awt.GridBagConstraints;
+import java.awt.GridBagLayout;
+import javax.swing.JButton;
+import javax.swing.JComponent;
+import javax.swing.JLabel;
+import javax.swing.JPanel;
+
+@SuppressWarnings("checkstyle:MethodName")
+public class SourceDisplayPanelForm extends JComponent {
```

Comment:
The layout seems odd, but we can work on this later. Let's focus on getting the code that changes views working.